### ISSUE TITLE
[VT-111] Adding a task does not follow "task add ..." conventions

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#132]
_PeterN on 2015-01-09T23:16:43Z says:_ 

 Let's create a new task with a project and a tag. Intuitively it should go as follows:
1.- press "a"
2.- project:myproj +mytag -- Description of the task
3.- a new task was created, except it bears a description "project:myproj +mytag -- Description of the task"

The problem is that in the routine task_add, vit escapes everything what was written on the prompt by pair of double quotes and executes:
> task add "what was written on the prompt"

What should have happened instead is:
> task add what was written on the prompt

I attach a patch fixing this.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-01-09T23:16:43Z
Modified: 2015-03-02T04:28:10Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-01T00:51:41Z says:_ 

 I agree that this is a bug. I think this might be related to VT-114. I will take a look at this (and your patch) for 1.3, which I hope to get a beta of out soon.

Thank you for your patch, Peter!

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-01T00:52:34Z says:_ 

 Peter posted a patch on VT-111. Perhaps it fixes both of these issues.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-01T02:49:40Z says:_ 

 Peter, are you sure this patch is for this bug? If not, do you still have the patch for this bug? If you do, and you feel like writing a proper commit message (referencing this bug report) and adding an entry in the ChangeLog, that would be great.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-02T04:27:55Z says:_ 

 Peter, please repost your patch on the bug it is meant for so it does not get lost.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-02T04:28:10Z says:_ 

 Fixed at 1dec1d2f for 1.3.

**Comment by pbeckingham** [comment]
[VT-111_filterfix.patch.txt](https://github.com/scottkosty/vit/files/1755276/VT-111_filterfix.patch.txt)
